src/logistration/tests/RegistrationPage.test.jsx

```diff
@@ -113,6 +142,75 @@ describe('./RegistrationPage.js', () => {
     expect(root.text().includes('Test University')).toBe(true);
   });
 
+  it('should match url after TPA redirection', () => {
+    const authCompleteUrl = '/auth/complete/google-oauth2/';
+    store = mockStore({
+      ...initialState,
+      logistration: {
+        ...initialState.logistration,
+        registrationResult: {
```

Comment:
default values should work. Don't need to define again.
Please remove other test cases as well.

Reply:
I don't want to use the default values, I need the values to be `true`, that is why I have added it here in test